Makefile

```diff
@@ -11,7 +11,7 @@ all: audit test build
 
 .PHONY: audit
 audit:
-	go list -m all | nancy sleuth
+	go list -json -m all | nancy sleuth --exclude-vulnerability-file ./.nancy-ignore
```

Comment:
Why has "--exclude-vulnerability-file ./.nancy-ignore" been added ?
I thought the latest plan was to not have that anymore (as its not needed / wanted) ?
Its now not in to many other projects.
Please check with Neill.

Reply:
My fault. It was from my review. I didn't know we were not supposed to have that anymore.